www/src/pages/blog/tags.js

```diff
@@ -39,11 +35,8 @@ const POPULAR_TAGS = [
 const PopularTagGrid = styled.div`
   display: grid;
   grid-auto-rows: 1fr;
-  grid-template-columns: repeat(3, 1fr);
+  grid-template-columns: repeat(auto-fit, minmax(130px, 1fr));
```

Comment:
@fk, is there a theme variable you'd like for this instead?

Reply:
No (because it doesn't exist … yet. Should it? What do y'all think?)